PROJECT

```diff
@@ -0,0 +1,19 @@
+domain: ibm.com
+layout:
+- go.kubebuilder.io/v3
+plugins:
+  manifests.sdk.operatorframework.io/v2: {}
+  scorecard.sdk.operatorframework.io/v2: {}
+projectName: ibm-block-csi-operator
+repo: github.com/IBM/ibm-block-csi-operator
+resources:
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: ibm.com
+  group: csi
```

Comment:
shouldn't this be `block.csi`? could you provide relevant links in this PR description?

Reply:
yes it was auto-generated

Reply:
and I will add links in this PR description

Reply:
I think the example link [is deprecated](https://github.com/operator-framework/operator-sdk-samples/blob/e832cb108ddf9ed6c7a9bbca8478a1d181a38ceb/README.md#deprecated-operator-sdk-samples).
please use [one of these](https://github.com/operator-framework/operator-sdk/tree/v1.9.0/testdata/go) (not sure which one)

Reply:
done